<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DescriptiveStat</class>
 <widget class="QDialog" name="DescriptiveStat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Descriptive Statistics</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>191</width>
     <height>261</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose Variables...</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>211</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Dispersion</string>
   </property>
   <widget class="QCheckBox" name="checkBoxSdev">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Std. deviation</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxVariance">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Variance</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxRange">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Range</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxSEMean">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>S.E. mean</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxMaximum">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Maximum</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxMinimum">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Distribution</string>
   </property>
   <widget class="QCheckBox" name="checkBoxKurtosis">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Kurtosis </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxSkewness">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Skewness</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>General</string>
   </property>
   <widget class="QCheckBox" name="checkBoxMean">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mean</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxMedian">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Median</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxSum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Sum</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>R Descriptive</string>
   </property>
   <widget class="QCheckBox" name="checkBoxSummary">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Summary</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxFivenum">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Fivenum</string>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>290</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
